### DISCUSSION TITLE
How to load a non-default code highlighting theme?

### DISCUSSION THREAD

**Discussion by amca01** [chipzoller/hugo-clarity#252] [Q&A]
As hugo-clarity uses the highlight.js engine for code displaying, there are (according to its website) some 243 different styles to choose.  I prefer a light style to dark - but how can I set it in my config file?  In the example site, `style="monokai"` is commented out.  But when I choose a different style, my code fails to render properly: quite a lot of code is invisible.  That is, it's displayed in the same colour as the background.  

Should I be using another parameter than `style`?  Or is there another way to choose a highlighting theme?  Thanks very much.

**Comment by onweru** [comment]
@amca01, there's no such parameter. If you want to change that side of things, you basically need to fork the theme and edit it  to match your desired styles. If you have a stylesheet that matches the classes generated by the clarity theme. Please the readme. I think there's a file you can use to override those styles.

**Reply by amca01** [reply]
Thank you - I had indeed read through the Readme, but I wondered if I might have missed something.  Would it be a hard thing to enable highlighting style changes as a parameter?  It seems a pity to use an engine like highlight.js but prevent the use of its flexibility.  (But then, what would I know...)

**Reply by onweru** [reply]
@amca01, umh, we used highlight.js for a niche use-case we needed (it had great support for CloudFront yaml). I would be dishonest to say that I'm aware of everything else that it's capable of. Would it be easy? I really have no idea. It's an open-source project so we allocate time for it when our schedules allow. We would be happy to review any contributions (PR) though.

**Reply by amca01** [reply]
I like hugo-clarity as a theme (I switched to it from Blackboard, which didn't seem to be regularly updated).  Given that Hugo supports chroma out of the box, is there any way of turning highlight.js "off" in hugo-clarity, so as to simply use chroma?   I'm well aware that as an open-source project the developers just work on it either when they can, or when they want to.  So I'm not _expecting_ any changes to be made.  But I'd be happy of any advice myself!  Anyway, thanks for all your discussions and explanations so far.
